exercises/perfect-numbers/example.scala

```diff
@@ -0,0 +1,26 @@
+import NumberType.NumberType
+
+object PerfectNumbers {
+  def classify(n: Int): NumberType = {
+    var sumOfFactors = 0
```

Comment:
Again, my Scala-conscience forces me to recommend an FP solution instead. :)

And again, you may just forget about this comment. :)